docs/markdown/Contributions/development/contributor-setup.md

```diff
@@ -28,6 +28,11 @@ Step 2: Bootstrap the Rust engine
 
 Pants uses Rustup to install Rust. Run the command from <https://rustup.rs> to install Rustup; ensure that `rustup` is on your `$PATH`.
 
+You also need a suitable Python 3.9 interpreter. The [Python Standalone Builds](https://gregoryszorc.com/docs/python-build-standalone/main/) by Gregory Szorc should work.
+Avoid the Python interpreters from Ubuntu's deadsnakes PPA, they don't work with Pants. The python39 package from the Nix package manager has also caused problems for users in the past.
+
+You also need to have the protobuf compiler and LLVM clang installed. On Debian derivatives, these can be installed using `apt install clang protobuf-compiler`.
```

Comment:
So unfortunately PBS doesn't _quite_ work. See [the quirks](https://github.com/indygreg/python-build-standalone/blob/main/docs/quirks.rst#references-to-build-time-paths).

Namely, if you consume an sdist YMMV on if you'll be able to compile it.

Unfortunately the Pants repo has `hdrhistogram` in its `plugin`, and currently that's sdist-only and contains C code. 

Meaning we're squarely in quirk territory. And looks like you hit the quirk, because PBS was compiled with `clang`, which is why the Python build of `hdrhistogram` died looking for `clang`.

So, we probably don't want to suggest PBS until we support it well, which likely means either getting rid of `hdrhistogram` from `StatsAggregator` or helping them build wheels: https://github.com/HdrHistogram/HdrHistogram_py/issues/46

Reply:
OK, I've removed the suggestion to use PBS.